tabs/configuration.js

```diff
@@ -255,6 +270,32 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
             orientation_mag_e.val(SENSOR_ALIGNMENT.align_mag);
         }
         
+        // ESC protocols 
+        
+        var escprotocols = [
+            'PWM',
+            'ONESHOT125',
+            'ONESHOT42',
+            'MULTISHOT'
+        ];
+        
+        var esc_protocol_e = $('select.escprotocol');
+
+        for (var i = 0; i < escprotocols.length; i++) {
+            esc_protocol_e.append('<option value="' + (i+1) + '">'+ escprotocols[i] + '</option>');
+        }
+        esc_protocol_e.val(PID_ADVANCED_CONFIG.fast_pwm_protocol+1);
+        
+        
```

Comment:
@KiteAnton: Shouldn't this be gated by a check for version > 2.8.1-RC1? Otherwise it will display options not available to users of older firmware.